hadoop-ozone/common/src/main/proto/OzoneManagerProtocol.proto

```diff
@@ -1097,11 +1097,34 @@ message UpdateGetS3SecretRequest {
     required string awsSecret = 2;
 }
 
+message OMServiceId {
+    required string serviceID = 1;
+}
+
+/**
+  This proto is used to define the OM node Id and its ratis server state.
+*/
+message RoleInfo {
+    required string omNodeID = 1;
+    required string ratisServerRole = 2;
+}
+
+/**
+  This is used to get the Server States of OMs.
+*/
+message ServiceState {
+    repeated RoleInfo roleInfos = 1;
+}
+
 /**
  The OM service that takes care of Ozone namespace.
 */
 service OzoneManagerService {
     // A client-to-OM RPC to send client requests to OM Ratis server
     rpc submitRequest(OMRequest)
           returns(OMResponse);
+
+    // A client-to-OM RPC to get ratis server states of OMs
```

Comment:
Also just wondering, is there a reason to add a new RPC and not use the submitRequet pattern -- that is add the message to the OMRequest / OMResponse pattern?

Reply:
You are right. There is no need for new RPC. I thought we could keep the admin protocols separate from the client protocols. But I think we can just use the OMRequest for now. Updated it.